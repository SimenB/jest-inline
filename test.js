test("some test", () => {
  expect({}).toMatchSnapshot();
});
