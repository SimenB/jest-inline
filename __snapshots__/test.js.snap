// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`some test 1`] = `Object {}`;
